prediction_market_agent_tooling/deploy/agent.py

```
@@ -446,6 +414,19 @@
     ) -> None:
         self.update_langfuse_trace_by_market(market_type, market)
 
+        api_keys = APIKeys()
+
+        self.check_min_required_balance_to_trade(market)
+
+        if market_type.is_blockchain_market:
+            # Exchange wxdai back to xdai if the balance is getting low, so we can keep paying for fees.
+            if self.min_balance_to_keep_in_native_currency is not None:
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)